common/plugins/com.liferay.ide.ui/plugin.xml

```diff
@@ -182,8 +182,8 @@
             id="com.liferay.ide.ui.workspace.launchWorkspace"
             name="Launch Workspace">
             <commandParameter
-                  id="com.liferay.ide.ui.workspace.launchWorkspace.workspaceLocation"
-                  name="Workspace Location"
+                  id="workspaceLocation"
```

Comment:
it is a bug  , the name should be the same as LaunchWorkspaceHandler.PARAM_WORKSPACE_LOCATION , otherwise the history workspace menu contribution will not show out